.agents/skills/synapse-a2a/references/commands.md

```
@@ -177,14 +211,17 @@ If `[REPLY EXPECTED]` marker is present, you **MUST** reply using `synapse reply
 
 ```bash
-# Use the reply command (--from is required in sandboxed environments)
+# Use the reply command (auto-routes to last sender)
+synapse reply "<your reply>"
+
+# In sandboxed environments (like Codex), specify your agent ID
 synapse reply "<your reply>" --from <your_agent_id>
 ```
 
 **Example - Question received (MUST reply):**
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)